### PR TITLE
Add tests for both version of JITHelpers.intrinsicIndexOf

### PR DESCRIPTION
**Pull request by dhong44**
Add tests for both versions of JITHelpers.intrinsicIndexOf.

Signed-off-by: Daniel Hong <daniel.hong@live.com>

### CODE REVIEW

**Comment by dhong44** [comment]
@fjeremic

File: test/functional/Java8andUp/src/org/openj9/test/com/ibm/jit/Test_JITHelpers.java [hunk 1]
```diff
@@ -102,6 +102,8 @@ public static void test_getSuperclass() {
 		0x57, 0x0, 0x58, 0x0, 0x59, 0x0, 0x5a, 0x0, (byte)0xc0, 0x0, (byte)0xc1, 0x0, (byte)0xc2, 0x0, (byte)0xc3, 0x0, (byte)0xc4, 0x0, (byte)0xc5,
 		0x0, (byte)0xc6, 0x0};
 
+	private static int[] indexes = new int[]{0, 1, 4, 7, 8, 9, 15, 16, 17, 31, 32};
```
Comment (fjeremic) [inline test/functional/Java8andUp/src/org/openj9/test/com/ibm/jit/Test_JITHelpers.java]: Can we fix indentation?

Comment (dhong44) [reply test/functional/Java8andUp/src/org/openj9/test/com/ibm/jit/Test_JITHelpers.java]: Sure, fixed in e1f5d62

**Comment by dhong44** [comment]
Windows failures due to #2129

**Pull request merged**